AppCenter/AppCenter/Internals/Ingestion/MSHttpIngestion.m

```diff
@@ -119,6 +105,46 @@ - (void)setEnabled:(BOOL)isEnabled andDeleteDataOnDisabled:(BOOL __unused)delete
   }
 }
 
+#pragma mark - MSHttpIngestion
+
+- (NSURL *)buildURLWithBaseURL:(NSString *)baseURL apiPath:(NSString *)apiPath queryStrings:(NSDictionary *)queryStrings {
```

Comment:
No logic changes. Extracted it as a method for being used by child classes.

Reply:
👍 i like it much better extracted anyway